### ISSUE TITLE
Tile full-resolution climate pngs

### ISSUE THREAD

**Issue by ajrobbins** [WikiWatershed/model-my-watershed#1976]
PNGs available here: https://github.com/BiG-CZ/BiG-CZ-Portal/blob/master/ClimatologyGriddedData.md#full-resolution-styled-pngs-intended-for-tiling-or-use-as-is

We need to tile these and load them into S3 for visualization in BiG CZ.

**Comment by rajadain** [comment]
Commenting to subscribe to this thread and get email notifications.

**Comment by mmcfarland** [comment]
Note that these are single, full-geography images per month.  I'm not sure if they are georeferenced, but it may be easier to generate tiles from the already ingested tiffs that represent this data.  If so, we'll need to match (or confirm a change) in the color ramps that are in the detailed json of the above link.

**Comment by rajadain** [comment]
This is one of the images:

![image](http://data.nanoos.org/files/cz/mapoverlays/ppt_01.png)

It's pretty low-res (1464 x 647, so less than a non-retina MacBook screen). Are these the kind of images, or are there more sophisticated editions out there?

**Comment by rajadain** [comment]
Found that image from the link above.

**Comment by mmcfarland** [comment]
No, that is representative of the PNGs they've provided, they are explicitly not pyramided tiles.  

We can't evaluate how they were used in the existing application (http://portal.bigcz.org/Explorer) because they don't appear to be available any more.  We either reach out (@ajrobbins) to see if the overlays can be restored for reference or we just generate the tiles ourselves.  Given how coarse the tiffs are (I think ~1km or more cell size), we wouldn't need to create many zoom levels and just stretch out when the numbers get high.

**Comment by ajrobbins** [comment]
@emiliom the pyramided tiles don't appear to be in use on portal.bigcz.org. Would it be possible to restore those for our reference? If it's a high effort, we'll go ahead and generate tiles from the raw tiffs.

**Comment by emiliom** [comment]
@ajrobbins, the overlays were never deployed on http://portal.bigcz.org proper. But they were deployed in other, similar applications, including the Monitor-My-Watershed DRB pilot at http://www.wikiwatershed-vs.org. Look under the "Remote Sensing" button. Also, if it makes any difference (I doubt it), here's another application where they're deployed (under "Remote Sensing" and "Models"): http://viz.criticalzone.org/Christina

@lsetiawan can provide more information about the tiles, if needed.

Finally, regarding an earlier comment on this thread, about the resolution: yes, these are continental-scale datasets. 1km is actually pretty high resolution, at that scale :wink:. I *think* the PRISM climatology (air temp & precip) may be at a coarser resolution still.

**Comment by lsetiawan** [comment]
> Explaining how you used the full PNGs

The PNG's explained [here](https://github.com/BiG-CZ/BiG-CZ-Portal/blob/master/ClimatologyGriddedData.md) are ingested by our UW EIS tile server, then they get chopped up, the output then is pyramided tiles 256x256. 

To use the UW EIS Tiles, here are the URL’s. @ajrobbins said the url's weren't working, but I double checked and they work. Please replace `{blah}` with the appropriate variables.

Air Temp: http://coral.apl.washington.edu/eis-live/tiles/prism-climate-air-temp-pcol/{month}/v1/ValueLocList/{z}/{y}/{x}.png
Precip: http://coral.apl.washington.edu/eis-live/tiles/prism-climate-precip-pcol/{month}/v1/ValueLocList/{z}/{y}/{x}.png

For `month` variable, here are the keys: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec’]

**Comment by ajrobbins** [comment]
>the overlays were never deployed on http://portal.bigcz.org proper
Ah, thanks @emiliom !
